src/translations/screens/Nearby.ts

```diff
@@ -167,22 +167,22 @@ const NearbyTexts = {
       hasPassedAccessibilityLabel: (time: string) =>
         _(
           `Avgangen ${time} har trolig passert plattformen.`,
-          `Departure ${time} has likely left the plattform.`,
+          `Departure ${time} has likely left the platform.`,
         ),
       upcomingRealtimeAccessibilityLabel: (time: string) =>
         _(
-          `Kommende avgang estimert til ${time}.`,
-          `Next departure estimated in ${time}.`,
+          `Kommende avgang har sanntid på ${time}.`,
```

Comment:
Maybe just have `Kommende avgang sanntid ${time}` here? Then we reinforce the pattern that we say "sanntid" right before announcing the time if it's realtime. I don't think "har sanntid på" gives enough extra information to deviate from the pattern. As always, feel free to argue ;)